monitoring/etcd/etcd.go

```diff
@@ -183,7 +163,46 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 	e.mutex.Lock() // To protect metrics from concurrent collects.
 	defer e.mutex.Unlock()
 	if err := e.collect(ch); err != nil {
-		log.Errorf("Error collecting stats from ETCD: %v", err)
+		log.Errorf("error collecting stats from ETCD: %v", err)
+	}
+}
+
+// Member represents simplified ETCD member struct
+type Member struct {
+	// ID of etcd cluster member
+	ID string `json:"id"`
+	// Name of etcd cluster member
+	Name string `json:"name,omitempty"`
+}
+
+type Members struct {
```

Comment:
This seems to be unused outside of `getMembers`. You can define the type inside the function if it does leak outside.